suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -430,6 +430,20 @@ export const switchDuplicatedDevice = createThunk(
     },
 );
 
+// Sort devices by timestamp and put Portfolio Tracker device at the end.
+const sortDevices = (devices: TrezorDevice[]) => {
+    return sortByTimestamp([...devices]).sort((a, b) => {
```

Comment:
Would not be the timestamp sorting overriden by the device `id` sorting? If I'm not mistaken, ommiting `sortByTimestamp` at all will have the same result as your current implementation, since there will be never two devices with the same device id.

Reply:
apart from this the PR LGTM :+1:

Reply:
The sort is stable, the order is not overriden by the second sort. So it works as stated in the comment - it's sorted by timestamp but portfolio tracker device is the last one. Should I add a unit test?